Corona-Warn-App/src/test/java/testhelpers/extensions/JsonExtensions.kt

```diff
@@ -2,22 +2,29 @@ package testhelpers.extensions
 
 import com.google.gson.Gson
 import com.google.gson.GsonBuilder
+import com.google.gson.JsonArray
+import com.google.gson.JsonElement
 import com.google.gson.JsonObject
 import io.kotest.assertions.assertionCounter
 import io.kotest.assertions.collectOrThrow
 import io.kotest.assertions.eq.eq
 import io.kotest.assertions.errorCollector
 import okhttp3.mockwebserver.MockResponse
 
+private fun String.determineJsonType(): Class<out JsonElement> = when {
```

Comment:
kinda hate this but i have no better idea 😆

Reply:
Test only :smirk: